### PR TITLE
append to pythonpath instead of overwriting it

### PR DESCRIPTION
**Pull request by MarcoGorelli**

### CODE REVIEW

**Review by girip11 — CHANGES_REQUESTED** [review]

File: nbqa/__main__.py [hunk 1]
```diff
@@ -344,7 +344,7 @@ def _run_command(
         If third-party tool isn't found in system.
     """
     env = os.environ.copy()
-    env["PYTHONPATH"] = os.getcwd()
+    env["PYTHONPATH"] = env.get("PYTHONPATH", "") + os.getcwd()
```
[the remark this answers is not in the document]
Comment (MarcoGorelli) [reply nbqa/__main__.py]: thanks for explaining!

`rstrip(os.pathsep)`, I guess?

have made the update anyway, still need to figure out how to write a failing test

[the remark this answers is not in the document]
Comment (girip11) [reply nbqa/__main__.py]: > rstrip(os.pathsep), I guess?

Good catch!

[the remark this answers is not in the document]
Comment (girip11) [reply nbqa/__main__.py]: You could use the remove_comments module for testing the same. Add `remove_comments` module to `PYTHONPATH` and run the test (in addition to the fixture test case)

[the remark this answers is not in the document]
Comment (MarcoGorelli) [reply nbqa/__main__.py]: Yes, that works! Thanks, is it good to go now?

**Review by girip11 — APPROVED** [review]

**Pull request merged**